app/models/declaration.rb

```diff
@@ -53,6 +53,9 @@ class Declaration < ApplicationRecord
   validate :declaration_date_within_milestone
   validate :mentorship_period_belongs_to_teacher
 
+  scope :completed, -> { where(declaration_type: "completed") }
+  scope :billable, -> { where(payment_status: %w[eligible payable paid]) }
```

Comment:
Can we extract these to a constant `BILLABLE_PAYMENT_STATUS` to avoid getting into the ECF pattern of repeating them everywhere? We can also use the state machine helpers here too if we wanted:

```
with_payment_status(:eligible, :payable, :paid)
```